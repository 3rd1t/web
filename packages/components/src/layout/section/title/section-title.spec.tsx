import React from "react"
import { render } from "@testing-library/react"

import SectionTitle from "./section-title"

describe(" SectionTitle", () => {
  describe("with subtitle", () => {
    it("should render successfully", () => {
      const { baseElement } = render(<SectionTitle title="TITLE" subtitle="SUBTITLE" />)
      expect(baseElement).toBeTruthy()
    })
  })
  describe("without subtitle", () => {
    it("should render successfully", () => {
      const { baseElement } = render(<SectionTitle title="TITLE" />)
      expect(baseElement).toBeTruthy()
    })
  })
})
