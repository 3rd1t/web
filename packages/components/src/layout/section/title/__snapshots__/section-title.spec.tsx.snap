// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` SectionTitle with subtitle should render successfully 1`] = `
<body>
  <div>
    <div
      class=" lg:text-center"
    >
      <h3
        class="mt-2 text-3xl font-extrabold tracking-tight text-gray-900 sm:text-4xl sm:leading-10"
      >
        TITLE
      </h3>
      <p
        class="mt-3 text-lg text-gray-700"
      >
        SUBTITLE
      </p>
    </div>
  </div>
</body>
`;

exports[` SectionTitle without subtitle should render successfully 1`] = `
<body>
  <div>
    <div
      class=" lg:text-center"
    >
      <h3
        class="mt-2 text-3xl font-extrabold tracking-tight text-gray-900 sm:text-4xl sm:leading-10"
      >
        TITLE
      </h3>
      <p
        class="mt-3 text-lg text-gray-700"
      />
    </div>
  </div>
</body>
`;
